x-pack/test/apm_api_integration/tests/services/throughput.ts

```diff
@@ -8,10 +8,15 @@
 import expect from '@kbn/expect';
 import qs from 'querystring';
 import { first, last } from 'lodash';
+import moment from 'moment';
 import archives_metadata from '../../common/fixtures/es_archiver/archives_metadata';
 import { FtrProviderContext } from '../../common/ftr_provider_context';
 import { registry } from '../../common/registry';
 
+function removeInvalidYAxis({ y }: { y: number | null }) {
```

Comment:
Why is this function called `removeInvalidYAxis`? Maybe `removeNullCoordinates` or something similar is better.

Reply:
Perhaps use `isFiniteNumber` instead?

Reply:
I'm going to remove this and replace with https://github.com/elastic/kibana/pull/88946/files#diff-338cceb311b19305c8b57c7c2f88391066d018907737be9507fbe45c602ffd58R16 after the PR is merged if that's ok with you.

Reply:
Function removed.